WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/SiteCreationIntentsViewModel.kt

```diff
@@ -115,20 +121,21 @@ class SiteCreationIntentsViewModel @Inject constructor(
             updateUiState(
                     state.copy(
                             isAppBarTitleVisible = true,
-                            isHeaderVisible = false
+                            isHeaderVisible = false,
+                            content = fullItemsList
```

Comment:
One thing I noticed is that once you focus on the text input, you can never get back to the default list. I wonder if we could use the empty / non-empty query to go between the (filtered) full list and the default list instead? Perhaps the default list isn't too important, so please don't consider this a blocker.. just something I noticed that I couldn't "undo" focusing on the box. Wdyt?

Reply:
This feels weird to me too. It was discussed internally(refs `pc8HXX-cp-p2#comment-309` and `pbArwn-447-p2#comment-5230`) though and decided to move with this behavior.